packages/ketchup/src/components/kup-data-table/kup-data-table-declarations.ts

```diff
@@ -102,6 +102,12 @@ export interface CellsHolder {
     [index: string]: Cell;
 }
 
+export interface Obj {
+    t: string;
```

Comment:
Redundant, must be removed - should be used `KupObj` from kupobjects-declarations instead